<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kana2epub</class>
 <widget class="QWidget" name="kana2epub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>651</width>
    <height>378</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>651</width>
    <height>378</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 5px;</string>
  </property>
  <widget class="QLabel" name="img_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>378</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>651</width>
     <height>378</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>651</width>
     <height>378</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-redius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../VScode/python/py_test/7z_uncompress/src/wallpaper.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 1px;
            background-color: rgba(45, 26, 255, 20);
			color:rgb(86, 86, 86);}
        QPushButton:hover{
            background-color: rgba(45, 26, 255, 100);}
        QPushButton:pressed{
            background-color: rgba(205, 10, 155, 100);}</string>
   </property>
   <property name="text">
    <string>日本語</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 1px;
            background-color: rgba(45, 26, 255, 20);
			color:rgb(86, 86, 86);}
        QPushButton:hover{
            background-color: rgba(45, 26, 255, 100);}
        QPushButton:pressed{
            background-color: rgba(205, 10, 155, 100);}</string>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 1px;
            background-color: rgba(45, 26, 255, 20);
			color:rgb(86, 86, 86);}
        QPushButton:hover{
            background-color: rgba(45, 26, 255, 100);}
        QPushButton:pressed{
            background-color: rgba(205, 10, 155, 100);}</string>
   </property>
   <property name="text">
    <string>中 文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 1px;
            background-color: rgba(45, 26, 255, 20);
			color:rgb(86, 86, 86);}
        QPushButton:hover{
            background-color: rgba(45, 26, 255, 100);}
        QPushButton:pressed{
            background-color: rgba(205, 10, 155, 100);}</string>
   </property>
   <property name="text">
    <string>KANA2EPUB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>586</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 8px;
            background-color: rgba(255, 255,0, 255);}QPushButton:hover{
            background-color: rgba(205, 205, 15, 200);}
QPushButton:pressed{
            background-color: rgba(205, 205, 15, 255);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 8px;
            background-color: rgba(255, 0, 0, 255);}
QPushButton:hover{
            background-color: rgba(220, 0, 15, 180);}
QPushButton:pressed{
            background-color: rgba(240, 0, 15, 170);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,100);
            border-radius: 1px;
            background-color: rgba(45, 26, 255, 20);
			color:rgb(86, 86, 86);}
        QPushButton:hover{
            background-color: rgba(45, 26, 255, 100);}</string>
   </property>
   <property name="text">
    <string>Language</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义输出路径</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>320</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>356</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>status bar： please wait a moment ......</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(231, 46, 255, 232)</string>
   </property>
   <property name="text">
    <string>あなたとはうんめーてきなものを感じますね。</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(231, 46, 255, 232)</string>
   </property>
   <property name="text">
    <string>多分、わたしはあなたと会うために生まれてきたのだと思いますよ。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            border: 1px solid rgba(155,155,155,0);
            border-radius: 4px;
            background-color: rgba(0, 0, 0, 0);}
QPushButton:hover{
            background-color:rgba(255, 247, 24, 120);}
QPushButton:pressed{
            background-color: rgba(240, 0, 15, 170);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../VScode/python/py_test/7z_uncompress/src/open_folder_file_icon_219486.ico</normaloff>../../../VScode/python/py_test/7z_uncompress/src/open_folder_file_icon_219486.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
